news/views.py

```diff
@@ -599,3 +581,64 @@ def list_newsletters(request):
     active_newsletters = Newsletter.objects.filter(active=True)
     return render(request, 'news/newsletters.html',
                   {'newsletters': active_newsletters})
+
+
+def update_user_task(request, api_call_type, data=None, optin=True, sync=False):
```

Comment:
Yes. I think it was a mistake for it to be in utils as evidenced by the runtime import. Also it's clearly a view, it's just not one called by url. There are slight differences, but it's around the removal of subscriber like the rest.